lib/index.js

```diff
@@ -31,21 +31,21 @@ function Setup(T, { checkTypes , _env=[] }){
     }
 
     function mapConstrToFn(constr) {
-    return (
-        constr === String
-            ? T.String
-        : constr === Number
-            ? T.Number
-        : constr === Boolean
-            ? T.Boolean
-        : constr === Object
-            ? T.Object
-        : constr === Array
-            ? T.Array(T.Any)
-        : constr === Function
-            ? T.AnyFunction
-        : constr
-    )
+        return (
+            constr === String
+                ? T.String
+            : constr === Number
+                ? T.Number
+            : constr === Boolean
+                ? T.Boolean
+            : constr === Object
+                ? T.Object
+            : constr === Array
+                ? T.Array(T.Any)
+            : constr === Function
+                ? T.AnyFunction

```

Comment:
I like the ease of use of paldepind's library,  I think I'd eventually like the users to be able to get value out this library without necessarily being aware or understanding what sanctuary-def is, while at the same time encouraging them to explore that ecosystem.

So if I were to remove these mappings, I'd probably keep inline predicates as a compromise, and potentially export $.Number, etc.  Not sure what the best solution is for that problem.  If you have any good ideas, please let me know 😄